strawberryfields/ops.py

```diff
@@ -584,6 +556,7 @@ class Decomposition(Operation):
     into a sequence of gates and state preparations.
 
     The first parameter p[0] of the Decomposition is always a square matrix.
+    TODO p[0] cannot be a symbolic parameter yet...
```

Comment:
should be put as a comment in code, not in docstring

Reply:
It's a restriction of the Decomposition class which I think cannot be easily overcome since we use various numerical matrix decompositions on `p[0]`, the user needs to be aware of it. I'll remove the words TODO and yet :)